mevolib/inference/__init__.py

```diff
@@ -174,3 +175,43 @@ def get_phylogeny(
         # Return the resultant phylogeny as a Bio.Phylo.BaseTree object and its
         # log-likelihood score
         return phylogeny, score
+
+def main():
+    """Default call for Inference module."""
+    parser = argparse.ArgumentParser(
+        description="Infers the phylogeny of a given set of aligned genes using the given inference"
+        + " tool and arguments"
+    )
+    parser.add_argument(
+        "-t", "--tool", required=True, help="Name or path of the phylogenetic inference tool."
+    )
+    parser.add_argument("-i", "--input", required=True, help="Aligned sequences input file")
+    parser.add_argument("-if", "--informat", required=False, help="Input file format (fasta, by default)")
+    parser.add_argument(
+        "-a",
+        "--args",
+        required=False,
+        help="Keyword or arguments to use in the call of the phylogenetic"
+        + " inference tool, excluding infile and outfile arguments. By default, 'default ' arguments are used.",
+    )
+    parser.add_argument("-o", "--output", required=True, help="Output file name (without extension)")
+    parser.add_argument("-of", "--outformat", required=False, help="Output file format. By default, 'NEWICK' format.")
+    parser.add_argument(
+        "-b",
+        "--bootstraps",
+        required=False,
+        help="Number of bootstraps to generate. By default, 0 (only uses the input alignment).",
+    )
+    args = parser.parse_args()
+
+    # We split the input file to obtain its name
+    filename = Path(args.input).stem
+    out_format = "newick" if args.outformat is None else args.outformat
+    get_phylogeny(
+        binary=args.tool,
+        infile=args.input,
+        infile_format="fasta" if args.informat is None else args.informat,
+        args="default" if args.args is None else args.args,
+        outfile=f"{filename}_inference.{out_format}",
+        outfile_format=out_format,
```

Comment:
```suggestion
    get_phylogeny(
        binary=args.tool,
        infile=args.input,
        infile_format=args.informat,
        args=args.args,
        outfile=f"{filename}_inference.{out_format}",
        outfile_format=args.outformat,
```
It is usually preferred to encode most of the logic in the argument parser itself (unless it cannot be done easily).